__tests__/suggestions.test.js

```diff
@@ -206,4 +206,19 @@ describe('Suggestions', function() {
 
     expect($el.componentDidUpdate.called).to.equal(true);
   });
+
+  test('should render custom suggestions', function() {
```

Comment:
`should render custom suggestions when renderSuggestion prop is provided`

Reply:
done